app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/mostPopularPatternsControls.jsx

```diff
@@ -128,7 +128,7 @@ export class MostPopularPatternsControls extends Component {
             hintType={'top-right'}
           />
         </FieldProvider>
-        <ModalField label={intl.formatMessage(messages.attributeKeyFieldLabel)} labelWidth={145}>
+        <ModalField label={intl.formatMessage(messages.attributeKeyFieldLabel)} labelWidth={120}>
```

Comment:
Please use `FIELD_LABEL_WIDTH` here from the `pages/inside/dashboardItemPage/modals/common/widgetControls/controls/constants.js`

Reply:
Done.